config/enums.yml

```diff
@@ -70,6 +70,10 @@
 #   pry>ENUMS::CAMPAIGN_STATUSES[2]          # active
 #
 
+BLOCK_LIST:
+  - https://cstream1.com/
+  - https://blockedurl.test/
+
```

Comment:
Might be better to list the domains then parse the referrer and check the domain.